product_sold_by_delivery_week/README.rst

```diff
@@ -57,7 +57,7 @@ info in other views like product recommendations, this is not needed.)
 Usage
 =====
 
-Once configured, hints will apperar as an optional column in sale order
+Once configured, hints will aperar as an optional column in sale order
```

Comment:
:thinking: I see I'm not the only typo blind in the house :smile: 
```suggestion
Once configured, hints will appear as an optional column in sale order
```

Or maybe we could just use *show up*

Reply:
haah ooops